File: client/src/app/_dtos/admin/userHistoryHeaderDto.ts
export interface IUserHistoryHeaderDto
{
	id: number;
	categoryRefCode: string;
	categoryRefName: string;
	customerName: string;
	completeBy: Date;
	assignedToName: string;
	assignedByName: string;
	assignedOn: Date;
	status: string;
	concluded: boolean;
	totalCount: number;
	totalNotContactible: number;
	totalContacted: number;
	totalPositive: number;
	totalNegative: number;

}

File: api/DTOs/HR/UserHistoryHeaderDto.cs
namespace api.DTOs.HR
{
    public class UserHistoryHeaderDto
    {
        public int Id { get; set; }
        public string CategoryRefCode { get; set; }
        public string AssignedToName { get; set; }
        public string AssignedByName { get; set; }
        public DateTime AssignedOn { get; set; }
        public DateTime CompleteBy { get; set; }
        public string Status { get; set; }
        public bool Concluded {get; set;}
        public int totalCount { get; set; }
        public int TotalNotContactible { get; set; }
        public int TotalContacted { get; set; }
        public int TotalPositive { get; set; }
        public int TotalNegative { get; set; }
    }
}